docs/guidance/autoscaler.md

```diff
@@ -63,21 +62,22 @@ Demands:
 
 #### Notes
 
-1. The operator will recognize the following setting and automatically inject a preconfigured autoscaler container to the head pod.
+1. To enable autoscaling, set your RayCluster CR's `spec.enableInTreeAutoscaling` field to true.
+   The operator will then automatically inject a preconfigured autoscaler container to the head pod.
    The service account, role, and role binding needed by the autoscaler will be created by the operator out-of-box.
    The operator will also configure an empty-dir logging volume for the Ray head pod. The volume will be mounted into the Ray and
    autoscaler containers; this is necessary to support the event logging introduced in [Ray PR #13434](https://github.com/ray-project/ray/pull/13434).
 
     ```
     spec:
-      rayVersion: 'nightly'
       enableInTreeAutoscaling: true
     ```
 
-2. The autoscaler image is `rayproject/ray:a304d1` which reflects the latest changes from the Ray master branch.
+2. If your RayCluster CR's `spec.rayVersion` field is at least `2.0.0`, the autoscaler container will use the same image as the Ray container.
+   For Ray versions older than 2.0.0, the image `rayproject/ray:2.0.0` will be used to run the autoscaler.
 
-3. Autoscaling functionality is supported only with Ray versions at least as new as 1.11.0. The autoscaler image used
-is compatible with all Ray versions >= 1.11.0.
+3. Autoscaling functionality is supported only with Ray versions at least as new as 1.11.0. Autoscaler support
+   is Beta as of Ray 2.0.0 and KubeRay 0.3.0
```

Comment:
```suggestion
   is beta as of Ray 2.0.0 and KubeRay 0.3.0
```
Do we have a link to what this means somewhere? Else just give a short description ("some APIs might change but should be mostly stable" or similar)

Reply:
Will add some words.

Reply:
Words added, thanks.